src/main/java/edu/byu/ece/rapidSmith/device/families/Kintexu.java

```diff
@@ -0,0 +1,719 @@
+package edu.byu.ece.rapidSmith.device.families;
```

Comment:
We should add the copyright disclaimer to the header of this file.  Just do it manually for now.  We have an outstanding issue to do it by default.